app/views/shared/_dap_analytics.html.erb

```diff
@@ -1,2 +1,8 @@
 <!-- <%= t('notices.dap_participation') %> -->
-<script async src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&subagency=TTS" id="_fed_an_ua_tag"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=<%= IdentityConfig.store.dap_measurement_id %>"></script>
```

Comment:
escape the url param just in case?
```suggestion
<script async src="https://www.googletagmanager.com/gtag/js?id=<%= CGI.CGI.escapeURIComponent(IdentityConfig.store.dap_measurement_id) %>"></script>
```

Reply:
Yep 👍 Updated in 889c98d.